README.md

```diff
@@ -11,36 +11,8 @@ Supported clouds are:
 *  [Docker](https://docs.docker.com/)
 
 ## Usage
-To see available options run ./scripts/cdperf --help
-
-```shell
-$ # build via docker
-$ make clean build
-$
-$ # alternatively, build locally (requires node + yarn installed)
-$ make local
```

Comment:
we have package.json in the project.  should we run `yarn && yarn build` ?

Reply:
no nothing needed, i've decided to make the transpiled scripts a part of this repo.